### PR TITLE
[Merged by Bors] - Add "fluvio connector logs" to CLI

### LINKED ISSUE #1926: Docs/Script for reading managed connector logs
For now, we shall just exec to the equivalent of `kubectl logs -f <pod name>`

### PR DESCRIPTION
**Pull request by simlay**
Closes #1926.

### CODE REVIEW

File: crates/fluvio-cli/src/connector/logs.rs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+            .args(["get", "pods", "-o=jsonpath='{.items[*].metadata.name}"])
+            .output()?
+            .stdout;
+        let pods = String::from_utf8_lossy(&pods);
+        let pod = pods.split(' ').find(|pod| pod.starts_with(&self.name));
+
+        if let Some(pod) = pod {
+            println!();
+            Command::new("kubectl")
+                .args(["logs", "-f", pod])
```
Comment (simlay) [inline crates/fluvio-cli/src/connector/logs.rs:33]: the `-f` should probably be optional.

**Review by sehz — APPROVED** [review]

**Comment by sehz** [comment]
add to changelog as well

**Pull request closed without merging**